src/main/java/bio/terra/stairway/FlightDao.java

```diff
@@ -300,7 +301,17 @@ private void disownRecoveryInner(String stairwayId)
       List<FlightContext> flightList = new ArrayList<>();
       try (ResultSet rs = getStatement.getPreparedStatement().executeQuery()) {
         while (rs.next()) {
-          FlightContext flightContext = makeFlightContext(connection, rs.getString("flightid"), rs);
+          String flightId = rs.getString("flightid");
+          FlightContext flightContext;
+          try {
+            flightContext = makeFlightContext(connection, flightId, rs);
+          } catch (JsonConversionException e) {
+            logger.error(
+                String.format("Unable to make FlightContext to disown flight id %s", flightId));
```

Comment:
nit: you don't need String.format with the logger

The wording is confusing me. Are you disowning the flight because you can't make a context, or failing to disown it?

Reply:
done thanks. I get messed up about it not working when you want to format and log the exception....